### ISSUE TITLE
Optimize View/Grid for small window sizes

### ISSUE THREAD

**Issue by foorschtbar** [o0shojo0o/PixelIt#71]
### Looks good:
![2021-09-12 19_38_12-PixelIt the Matrix Display PixelIt-Big](https://user-images.githubusercontent.com/10727275/132997292-5c6849d3-1146-473a-8140-6f77ef584004.png)

### Dark background behind "The Matrix Display" not aligned:
![2021-09-12 19_38_32-PixelIt the Matrix Display PixelIt-Big](https://user-images.githubusercontent.com/10727275/132997293-ef88ec8d-b2e0-4f2b-9fe3-2998748decc7.png)

### Menu missing. Burger Menu?
![2021-09-12 19_38_55-PixelIt the Matrix Display PixelIt-Big](https://user-images.githubusercontent.com/10727275/132997294-e5c8ed82-8f9d-42b1-94dc-4b55548dbbc6.png)

**Comment by o0shojo0o** [comment]
Yes the bug is known, I am currently testing with another framework to create a new web interface.
